ver/us/symbol_addrs.txt

```diff
@@ -18162,7 +18162,7 @@ D_80082358 = 0x80082358; // type:data
 D_80082388 = 0x80082388; // type:data
 gItemTable = 0x800878E0; // size:0x2D80
 D_8008A680 = 0x8008A680; // type:data
-D_8008D2C0 = 0x8008D2C0; // type:data
+dead_gItemTable = 0x8008D2C0; // type:data
```

Comment:
you can mark this dead for splat by doing dead:true I think? there's other examples in the file